### PR TITLE
Feature survey testing notes

### PR DESCRIPTION
**Pull request by hvyavuno**
```
 "lineItems": [{
        "extLineItemId": "en_US_lineItem",
        "surveyTestingNotes": "hello world",
        "_title": "US College",

```

### CODE REVIEW

**Comment by dynata-bradley** [comment]
The `.DS_Store` file included in this PR should be deleted.

File: lib/line_item.go [hunk 1]
```diff
@@ -212,6 +214,7 @@ type UpdateLineItemCriteria struct {
 	SurveyURLParams     []*URLParameter    `json:"surveyURLParams" valid:"optional"`
 	SurveyTestURLParams []*URLParameter    `json:"surveyTestURLParams" valid:"optional"`
 	Sources             *[]*LineItemSource `json:"sources,omitempty" valid:"optional"`
+	SurveyTestingNotes  *string           `json:"surveyTestingNotes,omitempty" valid:"optional"`
```
Comment (dynata-bradley) [inline lib/line_item.go]: Can you add test coverage for these changes to `line_item.go`?

Comment (hvyavuno) [reply lib/line_item.go]: Ok.

Comment (hvyavuno) [reply lib/line_item.go]: How do I test?

**Pull request merged**